src/leiningen/new/re_frame/.github/workflows/test.yaml

```diff
@@ -0,0 +1,26 @@
+name: Run tests
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: Tests
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: docker://mkapiczy/docker-java-node-chrome:latest
```

Comment:
https://hub.docker.com/r/mkapiczy/docker-java-node-chrome doesn't look well maintained. I'd prefer to use https://github.com/day8/dockerfile-for-dev-ci-image except that depends on Chrome 56.0.2924.0 (for commercial reasons). I'd like to split that image into two derivative images - one for latest Chrome and the other for Chrome 56.0.2924.0. I'll get that done then we can depend on the one with the latest Chrome here.

Reply:
Sounds great, can't wait for it !

Reply:
Docker image is ready. I'll make the edits in additional commit(s) after merging.